templates/default/rebuild-iptables.erb

```diff
@@ -130,7 +133,8 @@ end
 
 system_files = %w(/etc/debian_version /etc/redhat-release /etc/system-release)
 if system_files.any? { |file| File.exist?(file) }
-  install_rules(iptables_rules)
+  success = install_rules(iptables_rules)
+  raise "#{0}: failed to install iptables rules" unless success
```

Comment:
Nope I believe thats the correct syntax. `$0` is a *global* variable containing the name of the executable.